src/components/Table/Table.tsx

```diff
@@ -51,6 +51,47 @@ export class Table extends React.PureComponent<TableProps, TableState> {
   wrapperNode: HTMLElement
   tableNode: HTMLElement
 
+  setWrapperNode = node => {
+    this.wrapperNode = node
+    this.props.wrapperRef(node)
+  }
+
+  setTableNode = node => {
+    this.tableNode = node
+    this.props.tableRef(node)
+  }
+
+  getComponentClassNames = () => {
+    const { className, tableClassName, isLoading, onRowClick } = this.props
+    const { isTableCollapsed } = this.state
+
+    const tableWrapperClassNames = classNames(
+      `${TABLE_CLASSNAME}__Wrapper`,
+      isLoading && 'is-loading',
+      isTableCollapsed && 'is-collapsed',
+      className
+    )
+    const tableClassNames = classNames(
+      TABLE_CLASSNAME,
+      isLoading && 'is-loading',
+      Boolean(onRowClick) && 'with-clickable-rows',
```

Comment:
@tinkertrain What is this sorcery? I don't think I've ever seen `Boolean()` like that before, haha

Reply:
Whoa! What on earth is `Number('101')`!!!!
Learning so many new things! Hahaha

Reply:
Sorry for assuming...! `Number` and `Boolean` are both constructors for primitives in JS (you also have `String` and maybe I'm forgetting more), so you can declare a number in any of these ways:

```
var a = new Number('123')
var b = Number('123')
var c = 123
```

It's not very common to use, except sometimes without new.

From MDN
> In a non-constructor context (i.e., without the new operator), Number can be used to perform a type conversion.